tests/test_overlandflow.py

```diff
@@ -1,9 +1,13 @@
+import os.path
 import numpy as np
 import pfspinup.pfio as pfio
-from pfspinup.common import calculate_overland_flow
+from pfspinup.common import calculate_overland_flow, calculate_overland_flow_kinematic
```

Comment:
Should we name `calculate_overland_flow_kinematic` as `calculate_overland_flow` to indicate that's the recommended method, or just have one function (like in `R`) and make this algorithm the default?